_rules/auto-update-text-efbfc7.md

```diff
@@ -22,16 +22,21 @@ acknowledgements:
 
 The rule applies to any [visible][] [HTML element][] in an [HTML document][] if:
 
-- the `innerText` property of the [element][html element] changes without the user [activating][activate] or [focusing][focus] any [element][html element] in the same [HTML document][];
+- the `innerText` property of the [element][html element] changes;
 - the [element][html element] does not have [children][child] whose `innerText` property is also changed;
 - the change happens after the [readiness][document readiness] of the [HTML document][] the [element][html element] belongs to is equal to "complete";
+- the change happens when the [last activation timestamp][] of the [window][] the [HTML document][] is associated with has the value "positive infinity";
 - it is not the only [content][] in the [HTML document][].
 
-## Expectation
+## Expectation 1
 
-For the test target a [user interface component][] is provided to pause, stop or hide the change of the [text content][].
+For the test target a [user interface component][] is provided to pause, stop, hide or alter the frequency of the changes of the [visible text content][].
```

Comment:
The enumeration does not read very well due to the alter part having a specific part ("the frequency"). Makes it look like we could "stop the frequency" which does not make sense… (and "hide" refers to the content, while "pause" and "stop" refer to the changes and "alter" to "the frequency of the changes").

Also, I would replace "is provide" by a "there exists" formulation.